tests/urls.py

```diff
@@ -7,7 +7,8 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^', include('owm_legacy.urls', namespace='owm')),
+    url(r'^', include('owm_legacy.urls', namespace='owm_legacy')),
+    url(r'', include('openwisp_controller.urls')),
```

Comment:
ensure this replicates the url file of the latest ow-controller, which has changed a lot since this module was last updated